flow-launcher.tera

```diff
@@ -0,0 +1,201 @@
+---
+whiskers:
+  version: "2.3.0"
+  matrix:
+    - flavor
+  filename: "themes/Catppuccin {{ flavor.identifier | capitalize }}.xaml"
+---
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:system="clr-namespace:System;assembly=mscorlib">
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="pack://application:,,,/Themes/Base.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+    <Thickness x:Key="ResultMargin">0 0 0 8</Thickness>
+    <Style
+        x:Key="ItemGlyph"
+        BasedOn="{StaticResource BaseGlyphStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Foreground" Value="#{{ text.hex }}" />
+    </Style>
+    <Style
+        x:Key="QueryBoxStyle"
+        BasedOn="{StaticResource BaseQueryBoxStyle}"
+        TargetType="{x:Type TextBox}">
+        <Setter Property="FontSize" Value="24" />
+        <Setter Property="Foreground" Value="#{{ text.hex }}" />
+        <Setter Property="CaretBrush" Value="#{{ mauve.hex }}" />
+        <Setter Property="FontSize" Value="26" />
+        <Setter Property="Height" Value="42" /> 
+        <Setter Property="Padding" Value="0,0,0,0" />
+    </Style>
+    <Style
+        x:Key="QuerySuggestionBoxStyle"
+        BasedOn="{StaticResource BaseQuerySuggestionBoxStyle}"
+        TargetType="{x:Type TextBox}">
+        <Setter Property="Background" Value="#{{ mantle.hex }}" />
+        <Setter Property="Foreground" Value="#{{ surface1.hex }}" />
+        <Setter Property="FontSize" Value="26" />
+        <Setter Property="Height" Value="42" />
+        <Setter Property="Padding" Value="0,0,0,0" />
+    </Style>
+    <Style
+        x:Key="WindowBorderStyle"
+        BasedOn="{StaticResource BaseWindowBorderStyle}"
+        TargetType="{x:Type Border}">
+        <Setter Property="BorderThickness" Value="2" />
+        <Setter Property="BorderBrush" Value="#{{ if(cond=flavor.dark, t=surface0.hex, f=surface1.hex) }}" />
```

Comment:
I'd be interested in seeing how this actually affects the look of the application because it's actually quite hard to see what the `WindowBorderStyle` colour is from Pigeon's screenshots.

I'd imagine, after some testing, you could specify a colour that looks good both on all flavours, resulting in the removal of the if condition.